src/pages/my-profile/screens/MyProfileAdDetails/MyProfileAdDetailsTextArea/MyProfileAdDetailsTextArea.tsx

```diff
@@ -37,7 +31,14 @@ const MyProfileAdDetailsTextArea = ({ control }: TMyProfileAdDetailsTextAreaProp
                         value={value}
                     />
                 )}
-                rules={{ pattern }}
+                rules={{
+                    pattern: {
+                        message: localize(
+                            "Contact details can only include letters, numbers, spaces, and any of these symbols: -+.,'#@():;"
```

Comment:
i think u can use the getTextFielderror from utils

Reply:
i think we also need to fix that util function, its not going to translate the strings properly, plus we have to also pass in a localize('stringname') to make things work right

Reply:
we can't directly pass the string down as a prop to the function

Reply:
but for now ill use it

Reply:
yeah.. there are few more utils functions which are using the localize from deriv-com/translations instead of the hooks..